plugin/adsk/scripts/USDMenuProc.mel

```diff
@@ -13,8 +13,22 @@
 // limitations under the License.
 //
 
+int $_hasPrimUpdater = -1;
```

Comment:
I think we use the Maya standard naming for globals  which would be "$gXXX"

Reply:
Yeah, thought about that, but in this case we don't really want to expose this variable, it should not be accessed outside this source file.  I don't know if we have a naming convention for that.